bin/ck8s-kubespray

```diff
@@ -23,6 +23,8 @@ usage() {
     echo "      args: <prefix> [--extra-vars manual_prompt=true] [<options>]" 1>&2
```

Comment:
Shouldn't we have something in this binary to check if `ck8sKubesprayVersion` on your kubespray branch matches the one set in `CK8S_CONFIG_PATH`? I was expecting to see something like this

```
[ck8s] ERROR: Version mismatch. Run init to update config.
[ck8s] Config version: v2.20.0-ck8s3
[ck8s] CK8S version: 04b39733c17cd76370addae531e53db4616fe8a1
```
when running this command on mismatching versions
```
./bin/ck8s-kubespray run-playbook wc facts.yml
```

Reply:
Yeah, I can add something like that. I don't think `ck8s-kubespray init` is an idempotent command though, it's not really built to be run every time you do an upgrade like in apps. I could change that though

Reply:
I pushed a commit that adds this functionality, but I decided to not change how `init` works because making it so that it updates the version in the config may make the user expect it to work like apps init where it also makes other changes to the config. (Changing kubespray init to work just like apps init seems like it would be complicated because of how kubespray config differs from apps config, particularly the lack of default/override split.)